src/logics/Logic.cc

```diff
@@ -1308,9 +1310,9 @@ Logic::dumpHeaderToFile(ostream& dump_out) const
         Symbol const & symb = sym_store[s];
         dump_out << "(";
         for (SRef sr : symb) {
-            dump_out << getSortName(sr) << " ";
+            dump_out << sort_store.printSort(sr) << " ";
         }
-        dump_out << ") " << getSortName(symb.rsort()) << ")" << endl;
+        dump_out << ") " << sort_store.printSort(symb.rsort()) << ")" << endl;
```

Comment:
We could just call the `Logic`'s method, no?

```suggestion
        dump_out << ") " << printSort(symb.rsort()) << ")" << endl;
```

Reply:
Fixed.